src/gql/mutations/mod.rs

```diff
@@ -3,50 +3,50 @@ use graphql_client::GraphQLQuery;
 #[derive(GraphQLQuery)]
 #[graphql(
     schema_path = "src/gql/schema.graphql",
-    query_path = "src/gql/mutations/strings/PluginCreate.graphql",
+    query_path = "src/gql/mutations/strings/DeploymentRemove.graphql",
     response_derives = "Debug, Serialize, Clone"
 )]
-pub struct PluginCreate;
+pub struct DeploymentRemove;
 
 #[derive(GraphQLQuery)]
 #[graphql(
     schema_path = "src/gql/schema.graphql",
-    query_path = "src/gql/mutations/strings/PluginDelete.graphql",
+    query_path = "src/gql/mutations/strings/LoginSessionConsume.graphql",
     response_derives = "Debug, Serialize, Clone"
 )]
-pub struct PluginDelete;
+pub struct LoginSessionConsume;
```

Comment:
This is the actual addition. I reordered the mutations alphabetically in a [separate commit](https://github.com/railwayapp/cli/pull/339/commits/d860d697eb4281235f79c327672a3d4d49a202e6) so it is easier to decide where to put it. Sorry for a messy diff.